src/classes/STG_InstallScript.cls

```diff
@@ -222,6 +222,22 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
 
+     /*******************************************************************************************************
+    * @description Disables Preferred Phone handling when upgrading to 1.79
```

Comment:
Hmmm... we might need some changes here. We're releasing this in 1.80, not 1.79...

Reply:
And by 1.80 I mean 1.81 ;) (We should update the docs here too)